pages/patterns.html

```diff
@@ -0,0 +1,26 @@
+---
+# jekyll header
+---
+
+{% include header.html %}
+
+{% include container-start.html %}
+
+<div class="col-12">
+  <a name="patterns"> </a>
+  <h2>QI-Core - Patterns<a href="patterns.html#top" title="link to here" class="self-link"> <img src="target.png" width="20" class="self-link" height="20"/></a></h2>
+  <h4><a class="external free" href="http://hl7.org/fhir/us/qicore/StructureDefinition-qicore-observation.html" rel="nofollow">QI-Core - Observation</a> <a href="patterns.html#observation" title="link to here" class="self-link"> <img src="target.png" width="20" class="self-link" height="20"/></a></h3>
```

Comment:
This path should be relative ("StructureDefinition-qicore-observation.html")

Reply:
Done